config/resolvers/config-feature-flags.yaml

```diff
@@ -23,18 +23,10 @@ metadata:
     app.kubernetes.io/part-of: tekton-pipelines
 data:
   # Setting this flag to "true" enables remote resolution of Tekton OCI bundles.
-  # This is an experimental feature and thus should still be considered
-  # an alpha feature.
-  enable-bundles-resolver: "false"
+  enable-bundles-resolver: "true"
```

Comment:
Why are we changing this as part of making this feature `v1beta1`? 
I'm not entirely sure why they were disabled by default in alpha, but if we want to change that it feels like it should go in a different PR

Reply:
Ok, I can do that in a separate PR, if so desired. I wasn't a big fan of having them off by default in the first place, but the more I thought about it, the more I decided that it was a mistake to have them off. I just thought the `alpha`->`beta` transition was a good time to do that, particularly since we're also including the resolvers deployment in `release.yaml` in this PR. I'd need to make some changes to e2e test setup if I yank this out of the PR - let me know if you think I should.

Reply:
I think it's OK to make this swap in the same PR, but I'll defer to @afrittoli here. Just FYI that for v1 we're hoping to have resolution in beta and these resolvers (at least the cluster and bundle ones) enabled by default, as replacements for the OCI bundles syntax and ClusterTask. (ClusterTask might be less necessary since it's a separate CRD, but either way, I think we're waiting on https://github.com/tektoncd/pipeline/pull/5579 until this is merged.)

Reply:
I understand the reasoning for switching those flags by default, and since this PR is already baked (and rebased enough times) I think it's fine to keep them together - but generally I would prefer if we avoided grouping changes in one PR. I think a more natural flow would have been switching the flag default first and moving to beta then.

Reply:
👍 Will do in the future.